fontc/src/main.rs

```diff
@@ -14,9 +14,9 @@ use fontir::{
     stateset::StateSet,
 };
 use glyphs2fontir::source::GlyphsIrSource;
-use log::{debug, error, info, warn};
-use rayon::prelude::*;
+use log::{debug, error, info, log_enabled, trace, warn};
 use serde::{Deserialize, Serialize};
+use std::io::Write;
```

Comment:
nittiest nit nit: keep the std imports together, at the top.

Reply:
Fixed, ty